internal/host/installcmd.go

```diff
@@ -80,6 +80,21 @@ func (i *installCmd) GetStep(ctx context.Context, host *models.Host) (*models.St
 		data["INSTALLATION_TIMEOUT"] = strconv.Itoa(int(i.instructionConfig.InstallationTimeout))
 	}
 
+	if cluster.HTTPProxy != "" || cluster.HTTPSProxy != "" {
```

Comment:
Why this if is required if you check each proxy setting individually later?

Reply:
just to make sure we only provide the no-proxy if http-proxy or https-proxy are provided.
no-proxy by itself isn't useful.